src/Controller/Admin/AdministrationBaseController.php

```diff
@@ -133,10 +133,19 @@ public function index(): ?Response
         $this->set('propertiesSecrets', $this->propertiesSecrets);
         $this->set('metaColumns', $this->meta);
         $this->set('filter', []);
-        $this->set('schema', (array)$this->Schema->getSchema($this->resourceType));
+        $this->set('schema', []);
         $this->set('readonly', $this->readonly);
         $this->set('deleteonly', $this->deleteonly);
 
+        if ($this->resourceType === 'endpoint_permissions') {
```

Comment:
We should probably add these lines in `EndpointPermissionsController::index()` ?